content/enterprise/system-overview/capacity.mdx

```diff
@@ -39,6 +39,10 @@ system checks; it depends on the operator to provide enough memory to the system
 If the memory limit is adjusted to 1024 MB with the default capacity of 10, the instance would require, at a minimum,
 10 GB of memory reserved for Terraform runs.
 
+## Limiting CPU cores
+
+The default setting of `0` allows a Terraform run to utilize all available CPU cores on the TFE instance. You can adjust this setting by inputting a specific limit to the number of CPU cores the Terraform runs will have access to. 
```

Comment:
Thinking of this, could we tell folks where to adjust this? What's the name of the attribute or whatever that they need to set. I'm thinking we say "You can also set a value for `attribute_name` to specify a limit for the number of CPU cores Terraform runs can access." Or something similar. Thoughts?

```suggestion
The default setting of `0` allows a Terraform run to use all available CPU cores on the TFE instance. You can also input a limit for the number of CPU cores Terraform runs can access. 
```